packages/edit-site/src/components/global-styles/header.js

```diff
@@ -18,12 +18,7 @@ function ScreenHeader( { title, description, onBack } ) {
 			<View>
 				<Spacer marginBottom={ 0 } paddingX={ 4 } paddingY={ 3 }>
 					<HStack spacing={ 2 }>
-						<NavigatorToParentButton
-							style={
-								// TODO: This style override is also used in ToolsPanelHeader.
-								// It should be supported out-of-the-box by Button.
-								{ minWidth: 24, padding: 0 }
-							}
```

Comment:
I read [this comment](https://github.com/WordPress/gutenberg/pull/40592#discussion_r857652997) and it now looks like we can do without this override.